libs/components/src/lib/select/select.spec.ts

```diff
@@ -129,7 +129,11 @@ describe('vwc-select', () => {
 	});
 
 	describe('select icon', () => {
-		it('should add an icon to the select', async () => {
+		it('should not have the has-icon class by default', async () => {
```

Comment:
Try to be positive in your tests descriptions:
```suggestion
		it('should initialize without the "has-icon" class', async () => {
```

Reply:
Sure, updated it 👍